File: src/pl/file_read.pl
use Path::Class;

$contentFileStream = file("../files/content.txt");
print $contentFileStream->slurp() . "\n";

$multilineFileStream = file("../files/multipleLines.txt");
$fileHandle = $multilineFileStream->openr();
while($line = $fileHandle->getline()) {
    print "- $line";
}

File: src/erl/file_read.erl
-module(file_read).
-export([start/0]).

readAllLines(Device) ->
    case io:get_line(Device, "") of
        eof -> [];
        Line -> [Line] ++ readAllLines(Device)
    end.

start() -> 
    {ok, Data} = file:read_file("./../files/content.txt"),
    io:fwrite(Data),

    io:fwrite("~n"),

    {ok, Device} = file:open("./../files/multipleLines.txt", [read]),
    Lines = readAllLines(Device),
    lists:foreach(
            fun (Line) -> io:fwrite("- ~s", [Line]) end, 
            Lines),
    file:close(Device),

    erlang:halt().